gef-ui/server/controllers/facility-confirm-deletion/index.js

```diff
@@ -8,7 +8,7 @@ const facilityConfirmDeletion = async (req, res) => {
 
   try {
     const { details } = await api.getFacility(facilityId);
-    const heading = startCase(FACILITY_TYPE[details.type].toLowerCase());
+    const heading = startCase(FACILITY_TYPE[details.type.toUpperCase()].toLowerCase());
```

Comment:
Do we need to do this now, could it just be `FACILITY_TYPE[details.type]`?

Reply:
unfortunately, no. the facilityType has to be upper-cased because it matches the `key` from the constants.
i.e.
`CASH: 'Cash'`
and this `[details.type.toUpperCase()]` matches the `CASH` part

Reply:
Ahh I see, makes sense